From 4f31b6c9eee98240c978becd19f2bfea1b7e34d5 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Tue, 9 Jan 2018 01:53:07 +0200
Subject: [PATCH 1/1] Re-introduce build dir / dist dir option for isolated
 build environments.

This commit re-introduces the functionality for building lbry-app with a directory target. This is important for isolated build environments like solbuild, where build and install directories are different (and the contents of install dir are used in a final binary package) and thus necessitate being able to have unpackaged assets be placed in a directory relative to the build location, so instructions may be performed on those assets.
---
 package.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package.json b/package.json
index 2e61b99c..d0dc3f9c 100644
--- a/package.json
+++ b/package.json
@@ -20,6 +20,7 @@
     "dev": "electron-webpack dev",
     "compile": "electron-webpack && yarn extract-langs",
     "build": "yarn compile && electron-builder build",
+    "build:dir": "yarn build -- --dir -c.compression=store -c.mac.identity=null",
     "postinstall": "electron-builder install-app-deps",
     "precommit": "lint-staged",
     "lint": "eslint 'src/**/*.{js,jsx}' --fix",
-- 
2.15.1

